CONTRIBUTING.md

```diff
@@ -4,6 +4,24 @@ This document should concisely express the project development status,
 methodology, and contribution process.  As the community makes progress, we
 should keep this document in sync with reality.
 
+## Submitting a Pull Request (PR)
+
+The following outlines the general rules we follow:
+
+- All PRs must have the appropriate documentation changes made within the
+same PR. Note, not all PRs will necessarily require a documentation change
+but if it does please include it in the same PR so the PR is complete and
+not just a partial solution.
+- All PRs must have the appropriate testcases. For example, bug fixes should
+include tests that demonstrates the issue w/o your fix. New features should
+include as many testcases as possible to cover any variants of use of the
+feature.
+- PR authors will need to have CLA on-file with the Linux Foundation.
```

Comment:
...before PRs will be merged.  There is not a way to restrict people without the CLA signed from submitting a PR, AFAIK.